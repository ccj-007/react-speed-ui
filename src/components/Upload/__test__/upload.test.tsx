import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Upload } from '../../index';

describe('test Upload component', () => {
  it('should render', () => {
    const { asFragment } = render(<Upload action={''} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
