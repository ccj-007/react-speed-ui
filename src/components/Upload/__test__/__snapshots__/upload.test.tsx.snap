// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Upload component should render 1`] = `
<DocumentFragment>
  <div
    class="viking-upload-component"
  >
    <div
      class="viking-upload-input"
      style="display: inline-block;"
    >
      <button
        class="speed-button speed-button-primary css-1dis8g1-Button"
        data-testid="button"
      >
        Upload File
      </button>
      <input
        class="viking-file-input"
        style="display: none;"
        type="file"
      />
    </div>
    <ul />
  </div>
</DocumentFragment>
`;
